x-pack/plugins/cases/public/components/custom_fields/toggle/configure.test.tsx

```diff
@@ -30,24 +30,37 @@ describe('Configure ', () => {
     expect(screen.getByText(i18n.FIELD_OPTION_REQUIRED)).toBeInTheDocument();
   });
 
-  it('updates field options correctly', async () => {
+  it('updates field options correctly when not required', async () => {
     render(
       <FormTestComponent onSubmit={onSubmit}>
         <Configure />
       </FormTestComponent>
     );
 
-    userEvent.click(screen.getByText(i18n.FIELD_OPTION_REQUIRED));
+    userEvent.click(screen.getByText('Submit'));
+
+    await waitFor(() => {
+      // data, isValid
+      expect(onSubmit).toBeCalledWith({}, true);
+    });
+  });
 
+  it('updates field options correctly when required', async () => {
```

Comment:
Can you add a test where you change (interact) the `defaultValue`?

Reply:
I am adding a second test to still make sure in this one that the default is `false`.